tests/Components/CheckboxesTest.php

```diff
@@ -61,6 +61,31 @@ public function test_checkboxes_component_references_custom_hint()
         $view->assertSee('aria-describedby="favourite-flavour-hint"', false);
     }
 
+    public function test_checkboxes_reference_individual_hinst()
```

Comment:
It looks like there's a typo "hinst" instead of "hints"

Reply:
```suggestion
    public function test_checkboxes_reference_individual_hints()
```